digicampipe/utils/histogram.py

```diff
@@ -0,0 +1,52 @@
+from digicampipe.io.containers_calib import CalibrationHistogramContainer
+import numpy as np
+from histogram.histogram import Histogram1D
+
+
+def convert_histogram_to_container(histogram):
+    """
+    Utility function to convert an Histogram to a CalibrationHistogramContainer
+    :param histogram:
+    :return: CalibrationHistogramContainer
+    """
+
+    container = CalibrationHistogramContainer()
+
+    container.bins = histogram.bins.astype(np.int)  # TODO make ctapipe.HDFTableWriter accept unit32
+    container.count = histogram.data.astype(np.int)
+    container.shape = histogram.shape  # TODO need to accept tuple
+    container.n_bins = histogram.n_bins
+    container.name = histogram.name  # TODO need to accept str
+    container.axis_name = histogram.axis_name  # TODO need to accept str
+    container.underflow = histogram.underflow.astype(np.int)
+    container.overflow = histogram.overflow.astype(np.int)
+    container.max = histogram.max
+    container.min = histogram.min
+    container.mode = histogram.mode()
+    container.std = histogram.std()
+    container.mean = histogram.mean()
+
+    return container
+
+
+def convert_container_to_histogram(container):
```

Comment:
I think also this could be a method of the container ... `CalibrationHistogramContainer.to_histogram` or `as_histogram`.

Reply:
When it is a method of the cointainer, you do not need the type check for example ..

Reply:
Done